scripts/stage-release.sh

```diff
@@ -110,7 +109,7 @@ updateVersions() {
   local NEW_VERSION=$2
   mvn -q -B versions:set -DnewVersion="${NEW_VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
 
-  git add '**/pom.xml' 'pom.xml'
+  git add '**/*.yaml' '**/pom.xml' 'pom.xml'
```

Comment:
the `**/*.yaml` looks specific to the kroxylicious repo, we already diverge here since we don't have to do the `sed` work so I think we should remove it

Reply:
Fixed